src/shared/components/DurationInput.tsx

```diff
@@ -101,7 +103,11 @@ const DurationInput: FC<Props> = ({
         />
       </ClickOutside>
       {isFocused && (
-        <DropdownMenu className="duration-input--menu" noScrollX={true}>
+        <DropdownMenu
+          className="duration-input--menu"
+          noScrollX={true}
+          maxHeight={menuMaxHeight}
```

Comment:
since this is used in a couple of different places, I'd love to be confident that this doesn't introduce regressions to those other usages

Reply:
good point! i explicitly set a default value on line 45 for `menuMaxHeight`. The default value is from the [clockface doc](https://influxdata.github.io/clockface/?path=/story/components-dropdowns-family--dropdownmenu)  for `DropdownMenu`. i just double checked other places that use `DurationInput` has no affect on this change ✅